.github/workflows/build_and_release.yml

```diff
@@ -200,6 +200,85 @@ jobs:
           asset_name: ${{ env.ARTIFACT_NAME }}
           asset_content_type: application/octet-stream
 
+  # Create the AppImage
+  AppImage:
+    name: Create the AppImage
+    needs:
+      - determine_version
+      - create_release
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install the AppImage bundler and Performous deps
```

Comment:
This step can be made into a dockerfile. I think we'll be alright by just including the AppImageCrafter builder thingy and supplying a build-arg with it.

Reply:
It can't be because even building the AppImage has a dependency on `fuse` which is not straightforward to set up.